### DISCUSSION TITLE
What is your favorite Emoji 🥺

### DISCUSSION THREAD

**Discussion by ahaya3776** [NAIST-SE/AttractingContributors#59] [General]
Hi everyone 😎  
Tell me your favorite Emoji 🙏 🙏 🙏 
https://github.com/ahaya3776/AttractingContributors-emoji

**Comment by 278Mt** [comment]
added! and plz help my favorite movies https://github.com/278mt/AttractingContributors
thx;-)

**Comment by nosfill** [comment]
added!

**Comment by sergiodl-naist** [comment]
added one more!
Please visit and try to ask a question or recommend one vtuber on my fork!
https://github.com/sergiodl-naist/AttractingContributors-VTubers

**Comment by yokoro13** [comment]
I added!
Please help me!
https://github.com/yokoro13/AttractingContributors-VideoGames
